### ISSUE TITLE
Siteimprove is flagging 175th footer link as having the same text alternative

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#4399]
> The Accessibility Issue is "links in the same context with the same text alternative. Two or more links in the same context have the exact same link text — despite pointing to different pages. Vague or generic link text can be problematic for assistive technology users — especially if they're unable to see the visual context. It's better to provide a text alternative that can stand alone, clearly describing the destination or purpose of the link."
> 
> The links in question are the Iowa logos at the top and bottom of each page. The one in the header goes to uiowa.edu and the one in the footer goes to 175.uiowa.edu.

Related Cherwell Ticket 1213547

## Proposed Solution

Change the text associated with the footer svg.
